package.json

```diff
@@ -13,10 +13,10 @@
         "prestart:prod": "rimraf dist && npm run build",
         "start:prod": "node dist/main.js",
         "lint": "eslint --ext .js,.ts src/",
-        "test": "jest",
-        "test:watch": "jest --watch",
-        "test:cov": "jest --coverage",
-        "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+        "test": "NEW_RELIC_NO_CONFIG_FILE=true jest",
```

Comment:
Would it make sense for this to be in a config file?

Reply:
Probably but its not really relevant to this change so would need a new ticket, this was just added to remove error logging in the tests

Reply:
Fair enough. I'll create a new ticket